packages/components/src/range-control/README.md

```diff
@@ -152,6 +152,22 @@ If this property is true, a button to reset the the slider is rendered.
 - Required: No
 - Platform: Web | Mobile
 
+#### color
```

Comment:
What's the use cases for changing the color of the filled range? Wouldn't you just use CSS for this sort of thing?

Reply:
Halloo 👋 @ZebulanStanphill !

That's a good question :). For the components rendering Gutenberg/app UI (the `@wordpress/components` package), I believe customization should be done based on what a component exposes (it's API). 

Sort of like how a `<Button />`'s size can be adjusted with an `isSmall`, rather than using CSS for `height` and so on.

The UI can certainly be adjusted with CSS (any UI could really). But doing so is brittle, as selectors (usually classNames) are subject to change in the future.

In the case of the updated `RangeControl`, I've exposed the (track) colour, which can better accommodate upcoming UI updates (from the G2 initiative), as well as custom block developers.

That being said, we don't have to go with this implementation

Would love to hear what others think :)

Reply:
While I agree that we need a consistent way to change the color for different controls, this PR is the first that tries to introduce one. 

Options here could be a "theme" prop as well instead of very specific design props. 

So while I'm not against it in principle, I wonder if we should avoid adding these props for now and think about this kind of customization more globally?

Reply:
@youknowriad Oo! A `theme` prop sounds like a good API to encapsulate design props.

> I wonder if we should avoid adding these props for now and think about this kind of customization more globally?

I can omit the addition of this property for now. A global system for handling theming would be lovely, and it's something that [Emotion Theming](https://emotion.sh/docs/theming) supports. 

For instances where you may want to customize the aesthetic of a very specific, the `ThemeProvider` can be added to handle that.

Reply:
@youknowriad Update! For now, I've removed the `color` prop. We can approach theming in another PR.